Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:latest as builder
+
+WORKDIR ./shotover-proxy
+
+COPY ./ ./
+
+RUN cargo build
```

Comment:
I was wondering why the binary was 239MB,
you forgot to build in `--release`!

Reply:
Oops, yep I didn't want to wait as long in between tests.